kolibri/core/assets/src/views/tabs/index.vue

```diff
@@ -1,76 +1,33 @@
 <template>
 
-  <ui-tabs
-    ref="tabs"
-    class="tabs"
-    :type="type"
-    :disableRipple="true">
-    <ui-tab
-      v-for="item in items"
-      :title="item.title"
-      :icon="item.icon"
-      :selected="itemIsSelected(item)"
-      :disabled="itemIsDisabled(item)"/>
-  </ui-tabs>
+  <nav class="tabs">
+    <ul>
+      <slot/>
```

Comment:
I think I understand the reasoning behind this, but I can't help but think that this is a little too flexible if all we're going to be doing is putting in `tab-link`s. Was the intention to allow us to put in other items into this component? 

I might be missing something in my assumptions: is the purpose of using a slot to make the parent component more explicit? If so, this seems to be the perfect use case for a [scoped slot](https://vuejs.org/v2/guide/components.html#Scoped-Slots) :wink: . 

We could even break up the use cases to `hasIconOnly`, `hasTitleOnly`, and `hasBothTitleAndIcon`by having slots with different names.

Thoughts?